src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunityPage.java

```diff
@@ -0,0 +1,152 @@
+package salesforce.ui.pages.opportunity;
+
+import core.selenium.WebElementAction;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+import salesforce.utils.Translator;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CreatedOpportunityPage extends BasePage {
+    private final int sleepTime = 2000;
+
+    @FindBy(css = ".slds-theme--success span a div")
+    private WebElement textCreatedSuccessfull;
+
+    @FindBy(xpath = "//h1//slot//lightning-formatted-text")
+    private WebElement createdOpportunityTitle;
+
+    @FindBy(css = ".current + span")
+    private WebElement createdOpportunityStage;
+
+    @FindBy(xpath = "//a[@data-tab-value='detailTab']")
+    private WebElement detailBtn;
+
+    private By menuBtn = By.xpath("//li[contains(@class,'slds-dropdown-trigger_click')]");
+    private By deleteOption = By.xpath("//div[@role='menu']//*[@title='" + Translator.translateValue("Opportunity",
+            "delete") + "']//a");
+    private By deleteBtnConfirm = By.xpath("//button[@title=' + ConfigEnvVar.getResourceBundle()."
```

Comment:
Is this variable obtaining a value from an environment file?

Reply:
It was a mistake, an " was missing so I didn't notice, but I already fix it.